pkg/security/scc/byrestrictions.go

```diff
@@ -25,19 +26,29 @@ func (s ByRestrictions) Less(i, j int) bool {
 type points int
 
 const (
-	privilegedPoints points = 20
+	privilegedPoints points = 200000
 
-	hostVolumePoints       points = 10
-	nonTrivialVolumePoints points = 5
+	hostVolumePoints       points = 100000
+	nonTrivialVolumePoints points = 50000
 
-	runAsAnyUserPoints points = 4
-	runAsNonRootPoints points = 3
-	runAsRangePoints   points = 2
-	runAsUserPoints    points = 1
+	runAsAnyUserPoints points = 40000
+	runAsNonRootPoints points = 30000
+	runAsRangePoints   points = 20000
+	runAsUserPoints    points = 10000
+
+	runCapsDefaultPoints  points = 5000
+	runCapsAddOnePoints   points = 300
+	runCapsAllowAllPoints points = 4000
+	runCapsAllowOnePoints points = 10
```

Comment:
Seems counterintuitive that making a capability default weighs more heavily than making a capability allowed in the first place.  I would expect defaults to weigh little if they are to weigh anything at all.